File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x;
    // Ensure x is between 1 & 8
    do {
        x = get_int("Height:");
    } while(x > 8 || x <= 0);
    // Print pyramid
    for(int i=0; i < x; i++) {
        for(int j=x ; j > i+1; j--) {
            printf(" ");    
        }
        for(int k=0; k < i+1; k++) {
            printf("#");    
        }
        printf("  ");
        for(int k=0; k < i+1; k++) {
            printf("#");    
        }
        printf("\n");    
    }
}

File: mario.py
from cs50 import get_int

while True:
    x = get_int("Height:")
    if x <= 8 and x > 0:
        break

for i in range(x):
    print(" "*(x - i - 1), end="")
    print("#"*(i + 1), end="")
    print("  ", end="")
    print("#"*(i + 1), end="")
    print()